stakers-metrics/src/metrics.ts

```diff
@@ -96,10 +106,25 @@ async function collectSyncingMetric(network: typeof networks[number], type: "exe
   metric.set({ network: network }, isSyncing);
 }
 
+async function collectMevBoostMetric(network: typeof networks[number]) {
+  console.log(`Collecting MEV Boost metric for network: ${network}`);
```

Comment:
2 things:
- we use `logger` instead of console for logging purposes in this project
- there is no need to log anything while collecting the metric if there are no errors

Reply:
Done, logs deleted